independent-publisher-connectors/MyHospitals by AIHW/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2817 @@
+            "default": "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet",
+            "x-ms-visibility": "internal"
+          }
+        ]
+      }
+    },
+    "/reporting-units-downloads/{datasheet-code}/{reporting-unit-code}": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {}
+          }
+        },
+        "summary": "Get Reporting Unit Data Download",
+        "description": "Gets a reporting unit data download, in XLSX format, for the supplied reporting unit code and datasheet code.",
+        "operationId": "GetsReportingUnitDataDownload",
+        "parameters": [{
+            "name": "datasheet-code",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The datasheet code",
+            "x-ms-summary": "The datasheet code",
+            "default": "myh-reporting-unit",
+            "x-ms-url-encoding": "single",
+            "x-ms-visibility": "advanced"
+          }, {
+            "name": "reporting-unit-code",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The reporting unit code",
+            "x-ms-summary": "Reporting unit code",
+            "x-ms-url-encoding": "single",
+            "x-ms-dynamic-values": {
+              "operationId": "GetReportingUnits",
+              "value-path": "reporting_unit_code",
+              "value-collection": "result",
+              "value-title": "reporting_unit_name",
+              "parameters": {
+                "Accept": "application/json"
+              }
+            }
+          }, {
+            "name": "Accept",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet",
+            "x-ms-visibility": "internal"
+          }
+        ]
+      }
+    },
+    "/simple-downloads/download-codes": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {}
+          }
+        },
+        "summary": "Get Simple Downloads Available",
+        "description": "Gets dictionary, indexed by the simple download code, of all available simple download codes and descriptions that can be used in the simple download API call.",
+        "operationId": "GetSimpleDownloads",
+        "parameters": [{
+            "name": "Accept",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "application/json",
+            "x-ms-url-encoding": "single",
+            "x-ms-visibility": "internal"
+          }
+        ]
+      }
+    },
+    "/simple-downloads1/{download-code}": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {}

```

Comment:
@paulculmsee Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger.